design/src/production-chains/example-leafcutter.md

```diff
@@ -0,0 +1,38 @@
+# Example Production Chain: Leafcutter Ant Cultivation
+
+This document details a very simple production chain inspired by [leafcutter ants](https://en.wikipedia.org/wiki/Leafcutter_ant).
+
+## Step Zero: Acacacia Grows
+
+*Acacia* plants grow, producing *acacia leaves*.
+
+**Recipe:** 1 sunlight + 1 water + 1 soil nitrogen -> 1 acacia leaf. 10 second craft time.
+
+Plants automatically produce this recipe.
+Produced items are placed in their output.
+When the plant's output is full, it advances to the next **growth stage.**
```

Comment:
When is the output full (i.e., how many leaves are needed)?
How many growth stages are there?
Does the upgrade to the next growth stage cost time or is it instant?

Reply:
Also, if the output is harvested by the ants, the plant won't grow?
Can the plant "go back" a growth stage as the leaves are depleted?